src/contracts/atlas/Atlas.sol

```diff
@@ -327,7 +330,8 @@ contract Atlas is Escrow, Factory {
     /// @param solverOps SolverOperation array of the current metacall tx.
     /// @param returnData Return data from the preOps and userOp calls.
     /// @param key EscrowKey struct containing the current state of the escrow lock.
-    /// @return auctionWon bool indicating whether a winning solver was found or not.
+    /// @return auctionWon bool indicating whether a winning solver was found or not. key.solverOutcome is the index of
+    /// the winning solver. When no winner is found, key.solverOutcome is the error code of the last solver.
```

Comment:
I think tis line is intended to be just below, giving more details to the returned `EscrowKey`?

Reply:
fixed